.pre-commit-config.yaml

```diff
@@ -8,22 +8,23 @@ default_language_version:
 
 repos:
 - repo: https://gitlab.com/pycqa/flake8
-  rev: 3.8.3
+  rev: 3.8.4
```

Comment:
All changes to this file should probably not be part of this PR

Reply:
Hm, I'll open a separate PR.